passlib/apps.py

```diff
@@ -138,6 +138,14 @@ def _load_master_config():
 _django21_schemes.remove("django_bcrypt")
 django21_context = LazyCryptContext(schemes=_django21_schemes)
 
+# -----------------------------------------------------------------------
+# 3.1
+# -----------------------------------------------------------------------
+
+_django31_schemes = list(_django21_schemes)
+_django31_schemes.remove("django_pbkdf2_sha1")
```

Comment:
Shouldn't `django_pbkdf2_sha1` be added to a list of deprecated hashes so they could be migrated?

Reply:
Oh, I see that now. Yes, you're correct!